assets/src/application/ui/input/Button/ButtonGroup/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { Children, isValidElement } from 'react';
+
+export function cleanChildren(children) {
+	return Children.toArray(children).filter((child) => isValidElement(child));
+}
```

Comment:
handy little function...

but does this work?

```suggestion
	return Children.toArray(children).filter(isValidElement);
```

Reply:
updated based on suggestion

Reply:
this is copied from chakra: https://github.com/chakra-ui/chakra-ui/blob/master/packages/chakra-ui/src/utils/index.js#L184

Reply:
> but does this work?

what exactly seems tricky in this function?

Reply:
i didn't know if there was a reason you had chosen to do `filter((child) => isValidElement(child))` over `filter(isValidElement)`

so i wasn't 100% sure if there was something i was missing for why that latter suggestion wouldn't work

Reply:
I had a case in the past when it was important to keep this way:
```
filter((child) => isValidElement(child))
```
because when passed implicitly it will pass also the second argument of the filter which is index and if that function accepts the second argument, than it might mess things up

Reply:
checked `isValidElement` and it receives only 1 argument so we are good:

https://github.com/facebook/react/blob/master/packages/react/src/jsx/ReactJSXElementValidator.js#L66